eo-maven-plugin/src/test/java/org/eolang/maven/PlaceMojoTest.java

```diff
@@ -65,6 +65,14 @@ void placesBinaries(@TempDir final Path temp) throws Exception {
         );
     }
 
+    /**
+     * Test.
+     * @param temp Path to temp directory.
+     * @todo #1319:30min Why do we need lines 85-87 in this test?
```

Comment:
@OlesiaSub Those lines are set up for the mojo to write into known temp folders. Since mojo is instantiated manually (not by Maven) there will be no values for that props.

Reply:
@mximp ok but I still don't get why these lines are needed in this test, because the test passes if you remove them

Reply:
@OlesiaSub Event though the test passes, the files are written into a random place file system. We want it stay clean after the tests.

Reply:
@mximp oh ok got it now